main.go

```diff
@@ -58,20 +63,22 @@ var (
 )
 
 type FlagVar struct {
-	metricsAddr                 string
-	enableLeaderElection        bool
-	probeAddr                   string
-	rateLimiterFailureBaseDelay time.Duration
-	rateLimiterFailureMaxDelay  time.Duration
-	rateLimiterFrequency        int
-	rateLimiterBurst            int
-	reconciliationInterval      time.Duration
+	metricsAddr                       string
+	enableLeaderElection              bool
+	probeAddr                         string
+	rateLimiterFailureBaseDelay       time.Duration
+	rateLimiterFailureMaxDelay        time.Duration
+	rateLimiterFrequency              int
+	rateLimiterBurst                  int
+	reconciliationInterval            time.Duration
+	certificateReconciliationInterval time.Duration
```

Comment:
What'S the benefit of having this as param? How do we want to use this? Maybe it's better to keep this simple and just have it as a static value for now. Adding it as param will also make it relevant for OS documentation.

Reply:
I would not insist on having it as parameter, for me simpler would be also better will remove the configurable parameter. I do not believe we ever used it for the other reconciliations as well, but let keep them.